overlord/state/change.go

```diff
@@ -23,6 +23,17 @@ import (
 	"encoding/json"
 )
 
+// Status is used for status values for changes and tasks.
+type Status int
+
+// Admitted status values for changes and tasks.
+const (
+	Running Status = 0
```

Comment:
Works either way, but we _definitely_ don't want to change those values. That's being serialized into state!